src/server/MessageProcessor.js

```diff
@@ -0,0 +1,419 @@
+/**
+ *  Copyright (c) Facebook, Inc.
```

Comment:
I've double-checked this and I think we intentionally omitted the year range here